_includes/https_redirect.html

```diff
@@ -0,0 +1,8 @@
+<script>
+  // If we're running on a real web server (as opposed to localhost on a custom port,
+  // which is whitelisted), then change the protocol to HTTPS.
+  // See https://goo.gl/lq4gCo for an explanation.
+  if ((!location.port || location.port == "80") && location.protocol != 'https:') {
```

Comment:
can we just check `location.origin !== 'localhost'` so people can use their own port